package.json

```diff
@@ -2,7 +2,7 @@
   "name": "react-native-elements",
   "version": "3.2.0",
   "description": "React Native Elements & UI Toolkit",
-  "main": "src/index.js",
+  "main": "src/index.tsx",
```

Comment:
`main` should be some transpiled js so consumers aren't forced to transpile this module to use it

(i.e. `require('react-native-elements'))` _should_ work in node from a syntax perspective even if it relies on some globals only present in a React Native env

Reply:
There are definitely still some bugs to work out here before merging into next. If you have time to test it out and open a PR to fix this, it would be much appreciated.